lib/steady_state/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SteadyState
-  VERSION = '1.1.0'
+  VERSION = '1.2.0'
```

Comment:
I think you'll need to re-run `bundle install` and `bundle exec appraisal install` (and `bundle exec rubocop -A` if appraisal introduces linter errors, lol), and then commit those changes. That should be the last piece of this that will unblock your build.

Reply:
Thanks!  I started on this yesterday and got sidetracked - should be incoming!